app/views/accounts/show.html.erb

```diff
@@ -0,0 +1,53 @@
+<% content_for :title, "Settings" %>
+
+<% content_for :breadcrumbs,
+   render("govuk_publishing_components/components/breadcrumbs", {
+     collapse_on_mobile: true,
+     breadcrumbs: [
+       {
+         title: "Dashboard",
+         url: root_path,
+       },
+       {
+         title: "Settings",
+       }
+     ]
+   })
+%>
+
+<%= render "govuk_publishing_components/components/cards", {
+  items: [
+    {
+      link: {
+        text: "Change your email or password",
+        path: account_email_password_path,
+      },
+      description: "Reset your password or change your email address."
+    },
+    (
+      {
+        link: {
+          text: "Manage permissions",
+          path: edit_user_path(current_user),
+        },
+        description: "Make changes to your permissions across different apps.",
+      } if policy(current_user).edit?
```

Comment:
I stole this `policy` from the AccountsController but I'm not sure it's what I want here